src/pages/track/TrackPage.tsx

```diff
@@ -4,13 +4,17 @@ import {CheckpointType, TeamType} from "../../services/getTeamTokenMock.tsx";
 import {getCurrentCheckpoint} from "../../services/getCurrentCheckpointMock.tsx";
 import {sendAnswer} from "../../services/sendAnswerMock.tsx";
 import Loading from "../../components/Loading.tsx";
+import LogOut from "../../components/LogOut.tsx";
+import {logout} from "../../services/logOut.tsx";
 
 const emptyCheckpoint: CheckpointType = {question: "", answer: ""};
 
 type TrackProps = {
     team: TeamType,
+    setTeam: (team: TeamType | undefined) => void,
```

Comment:
From point of view of component they receive props and emit events. Maybe a better approach could be call this prop `onTeamChange`, instead of `setTeam`.

When you read a `setter` in a component you usually think that you are handling your own state and not a parent one.